snake_game_high_score/scoreboard.py

```diff
@@ -0,0 +1,35 @@
+from turtle import Turtle
+ALIGNMENT = "center"
+FONT = ("Courier", 24, "normal")
+
+
+class Scoreboard(Turtle):
+
+    def __init__(self):
+        super().__init__()
+        self.score = 0
+        with open("data.txt") as data:
+            self.high_score = int(data.read())
+        self.color("white")
+        self.penup()
+        self.goto(0, 270)
+        self.hideturtle()
+        self.update_scoreboard()
+
+    def update_scoreboard(self):
+        self.clear()
+        self.write(f"Score: {self.score} High Score: {self.high_score}", align=ALIGNMENT, font=FONT)
+
+    def reset(self):
+        if self.score > self.high_score:
+            self.high_score = self.score
+            with open("data.txt", mode="w") as data:
+                data.write(f"{self.high_score}")
+        self.score = 0
+        self.update_scoreboard()
```

Comment:
This function isn't working! See if you can troubleshoot why.

Reply:
I checked it and I figured out that the data.txt wasn't existing I dont know why.. do I need to fetch it or I can fix it directly here on Github.

Reply:
No need to actually address it! Unless the file is in the `.gitignore` (please check!) in which case you should either 
1. Remove it
2. Write code to generate it with a `0` in the file if it doesn't exist (better solution)